hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/transaction/lock/StorageBasedLockProvider.java

```diff
@@ -590,4 +612,45 @@ private void logErrorLockState(LockState state, String msg) {
   long getCurrentEpochMs() {
     return System.currentTimeMillis();
   }
+
+  /**
+   * Helper method to start audit operation when lock is acquired.
+   */
+  private void startAuditOperation(long timestamp) {
+    storageLpAuditService.ifPresent(auditService -> {
```

Comment:
nit: can u write a static method to do this one time?

private void withAuditService(Consumer<AuditService> action) {
    auditService.ifPresent(auditService -> {
        try {
            action.accept(auditService);
        } catch (Exception e) {
            logger.warn("Owner {}: Failed to update audit operation: {}", ownerId, e.getMessage());
        }
    });
}